### PR TITLE
feat(condition): add `isVisible` condition

### PR DESCRIPTION
**Pull request by marstamm**
related to https://github.com/bpmn-io/bpmn-js-element-templates/issues/19

### CODE REVIEW

File: packages/element-templates-json-schema-shared/src/defs/condition.json [hunk 1]
```diff
@@ -56,7 +56,18 @@
...
+          "properties": {
+            "isVisible": {
+              "type": "boolean"
+              
+            }
+          },
+          "required": [
+            "isVisible"
+          ]
+        },
```
Comment (barmac) [inline packages/element-templates-json-schema-shared/src/defs/condition.json:70]: Notice that trailing comma is not allowed in JSON.

File: packages/element-templates-json-schema-shared/src/defs/condition.json [hunk 2]
```diff
@@ -56,7 +56,18 @@
           "required": [
             "oneOf"
           ]
-        }
+        },
+        {
+          "properties": {
+            "isVisible": {
+              "type": "boolean"
+
```
Comment (barmac) [inline packages/element-templates-json-schema-shared/src/defs/condition.json:64]:
```suggestion
```

File: packages/element-templates-json-schema-shared/src/defs/condition.json (hunk 1 above)
Comment (barmac) [reply packages/element-templates-json-schema-shared/src/defs/condition.json:70]:
```suggestion
        }
```

**Comment by barmac** [comment]
Shall we add a description to the new field? I don't think it's self-explanatory from the name.

**Comment by barmac** [comment]
This could be something like:
> Activates property based on whether linked property is active.

Let's also not forget to document when `bpmn:Message#zeebe:subscription#property` of `name=correlationKey` is implicitly hidden ^^

**Comment by marstamm** [comment]
> Let's also not forget to document when bpmn:Message#zeebe:subscription#property of name=correlationKey is implicitly hidden ^^

Where would be the best place to do this? Just the description of the `name` property or do we need to be more specific?

**Comment by barmac** [comment]
Camunda Docs once this PR is merged: https://github.com/camunda/camunda-docs/pull/2923/

File: packages/element-templates-json-schema-shared/src/defs/condition.json [hunk 3]
```diff
@@ -56,6 +56,16 @@
           "required": [
             "oneOf"
           ]
+        },
+        {
+          "properties": {
+            "isVisible": {
+              "type": "boolean"
```
Comment (barmac) [inline packages/element-templates-json-schema-shared/src/defs/condition.json:63]:
```suggestion
              "type": "boolean",
              "description": "For `true`, activates the property when given property is active."
```

**Comment by barmac** [comment]
Regarding description, see: https://github.com/camunda/element-templates-json-schema/pull/120/files#r1400472914

File: packages/element-templates-json-schema-shared/src/defs/condition.json (hunk 3 above)
Comment (marstamm) [reply packages/element-templates-json-schema-shared/src/defs/condition.json]: added via [15a9a3b](https://github.com/camunda/element-templates-json-schema/pull/120/commits/15a9a3bf59e00466cf86a068cec69c12dd73ce3e)

**Comment by barmac** [comment]
Hmm maybe we call it `isActive` instead? So that it cannot be confused due to `Hidden` type.

**Review by barmac — APPROVED** [review]

**Pull request merged**